bin/pycbc_upload_xml_to_gracedb

```diff
@@ -58,6 +78,24 @@ parser.add_argument('--min-ifar', type=float, metavar='YEARS',
 parser.add_argument('--production-server', action="store_true", default=False,
                     help="Upload event to production graceDB. If not given "
                          "events will be uploaded to playground server.")
+parser.add_argument('--force-overwrite', action='store_true', default=False,
+                    help="GraceDb instance will be checked for if an event "
+                         "with the same event time and FAR already exist. "
+                         "If so, event will not be uploaded")
+parser.add_argument('--query-string', default='pycbc',
+                    help="If not using --force-overwrite, add a string to "
+                         "gracedb query to further filter events. "
+                         "Default='pycbc'")
+parser.add_argument('--filename-T050017',
+                    help="If given use T050017-v1 naming convention for "
+                         "output XML filename. Argument is string for "
+                         "search identifier in filename, e.g. 'AllSky' "
+                         "would give H1L1V1-PYCBC_AllSky-1234567890-1.xml.")
+# https://dcc.ligo.org/LIGO-T050017/public
```

Comment:
This comment probably goes before the previous `add_argument` call, or maybe even in the help string for the option?